StubsController.php

```diff
@@ -92,6 +92,8 @@ public function actionIndex()
                     $class = $component['class'];
                 } elseif (isset($component['__class'])) {
                     $class = $component['class'];
```

Comment:
@antonovsky you broke my previous pull request. It should be 

` $class = $component['__class'];`

Reply:
Sorry, i fix it here: https://github.com/bazilio91/yii2-stubs-generator/pull/21